### ISSUE TITLE
Handle AsciiDoc, not just Markdown

### ISSUE THREAD

**Issue by abingham** [abingham/accu-2017-elm-app#24]
We currently treat the proposal descriptions as Markdown, and this kinda-sorta works. However, it's apparently in AsciiDoc. It's somewhat amazing that the markdown parser does as good a job as it does on the current text!

Anyhow, see if there's anything to be done about this.

**Comment by abingham** [comment]
See perhaps [asciidoctor.js](http://asciidoctor.org/news/2013/05/21/asciidoctor-js-render-asciidoc-in-the-browser/)

We'll probably have to do something similar to [the current markdown implementation](http://package.elm-lang.org/packages/evancz/elm-markdown/latest) to integrate with elm.

**Comment by abingham** [comment]
There is (was?) [a bug](https://github.com/asciidoctor/asciidoctor.js/issues/178) that prevents webpack from working properly with asciidoctor.js. It's supposed to be fixed in 1.5.5, but I haven't been able to make it work yet. I get errors like this:
```
ERROR in ./src/static/asciidoctor.min.js
Module not found: Error: Cannot resolve module 'opal-runtime' in /Users/sixtynorth/projects/accu-2017-elm-app/src/static
 @ ./src/static/asciidoctor.min.js 1:94-117

ERROR in ./src/static/asciidoctor.min.js
Module not found: Error: Cannot resolve module 'fs' in /Users/sixtynorth/projects/accu-2017-elm-app/src/static
 @ ./src/static/asciidoctor.min.js 579:280-293 581:458-471 581:528-541 759:309-322
```

**Comment by abingham** [comment]
Some initial work in [the `ascii-doc` branch](https://github.com/abingham/accu-2017-elm-app/tree/ascii-doc).

**Comment by abingham** [comment]
Fixed in dcd40da18c4c12d90a0b0b26a6f4a043d713c992
